rllib/algorithms/algorithm_config.py

```diff
@@ -175,6 +175,7 @@ def __init__(self, algo_class=None):
         self.evaluation_interval = None
         self.evaluation_duration = 10
         self.evaluation_duration_unit = "episodes"
+        self.evaluation_sample_timeout_s = 180.0
```

Comment:
New config value. Triggers a meaningful warning if breached with various things to try to fix the timeouts.